tests/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019 Leiden University Medical Center
```

Comment:
Adding `__init__.py` in tests is not really necessary. Unless you want to test all files for correct typing in tests with mypy.